receipt_dynamo/fix_exceptions_single.py

```
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+"""
+Script to fix exception inconsistencies in a single receipt_dynamo data file.
+Usage: python fix_exceptions_single.py <filename>
+"""
+
+import re
+import sys
+from pathlib import Path
+from typing import Dict, List, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)